app/src/main/java/com/spentwell/adapters/DashboardAdapter.kt

```diff
@@ -37,20 +50,34 @@ class DashboardAdapter(
     }
 
     override fun onBindViewHolder(holder: RecyclerView.ViewHolder, position: Int) {
+
         if (holder is ExpenseCategoryViewHolder) {
+            val totalAllocationAmount = when(position) {
```

Comment:
Check for item.expenseType instead of position

Reply:
fixed @shankar75031